src/native-common/Image.tsx

```diff
@@ -109,6 +112,12 @@ export class Image extends React.Component<Types.ImageProps, Types.Stateless> im
         );
     }
 
+    componentWillReceiveProps(nextProps: Types.ImageProps) {
+        if (!_.isEqual(this.props, nextProps)) {
```

Comment:
Do we need to check all props to set `this._forceCache`?

Reply:
We can be selective here but it doesn't hurt to retry without ```force-cache``` on all prop changes to keep the code simple and future-proof.

Reply:
ok. props can include callbacks _(not references, `onLoad: () => {}`)_ and this conditions could fail, also `isEqual` performs a deep comparison between two values - so it can impact on performance. Just need to keep in mind that.

Reply:
Great point, thanks. The web implementation also compares only a few props so I've done the same here.